benchmarks/2019.json

```diff
@@ -2447,6 +2447,24 @@
       100
     ]
   },
+  {
```

Comment:
These (266 claims, EHR and registry) should be include according the past logs and current logs. Don't know if they were excluded for some other reason.

Reply:
Was not added/removed ever

Reply:
@sarahnw Could you help look into this please?

Reply:
Should be removed. From an email in last Nov/Dec, "This measure underwent significant revisions from 2017 to 2018 and now as you know has three strata" - CMS agreed to remove it from historic benchmarks since this would've used 2017 submitted measures. Please remove and good catch!

Reply:
Cool. For 226 cmsWebInterface, it was added manually. Should I remove that too or keep for some reason?
```
Preventive Care and Screening: Tobacco Use: Screening and Cessation Intervention,226,cmsWebInterface,Process,Y,0 - 55.22,55.22 - 61.76,61.76 - 68.18,68.18 - 73.85,73.85 - 79.55,79.55 - 85.67,85.67 - 92.31,>= 92.31,No
```